selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/editor/autocomplete/AutocompleteProposalJavaDocTest.java

```diff
@@ -206,41 +220,39 @@ public void shouldDisplayJavaDocOfJreClass() throws IOException {
   }
 
   @Test
-  public void shouldWorkAroundAbsentSourcesOfExternalLib() throws IOException {
-
-    // This test fails because jdt.ls does download source for the class being used here. Need to
-    // find or construct an artifact that has no source.
+  public void shouldNotShowJavaDocIfExternalLibDoesNotExist() throws IOException {
     // when
     editor.waitActive();
     loader.waitOnClosed();
     editor.goToCursorPositionVisible(31, 23);
     editor.launchAutocompleteAndWaitContainer();
-    editor.selectAutocompleteProposal("info(String msg) : void Logger");
+    editor.selectCompositeAutocompleteProposal("info(String msg) : void Logger");
 
     // then
-    editor.waitProposalDocumentationHTML(".*No documentation found.*");
```

Comment:
This defeats the purpose of the test. We need an example that does not have javadoc or source.

Reply:
So if we don't have "Download source" feature i suggest to leave this test as I reworked (now it checks when jDoc is empty and popup window doesn't shown).
And I've also created one issue to write new test: https://github.com/eclipse/che/issues/10981

Reply:
Sorry, I misread the test here...If the removal of the slf4j artifacts works, it should show "No documentation found" afterwards, no?
I would think that the code jar is still there because the file is opened and cannot be removed.

Reply:
Now quick documentation shows in Orion tooltip and when Language Server returns empty string - tooltip isn't shown. That is why no "No documentation found" here